<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>m3u8</class>
 <widget class="QWidget" name="m3u8">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
<<<<<<< HEAD
    <height>299</height>
=======
    <height>324</height>
>>>>>>> f380bd248a44e99d1dc87085ebda5125608c9d1f
   </rect>
  </property>
  <property name="windowTitle">
   <string>m3u8</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resource/logo.ico</normaloff>resource/logo.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
<<<<<<< HEAD
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEdit_m3u8">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>请输入m3u8网址</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>保存文件名：</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>下载路径：</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
=======
   <item row="11" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>确认下载</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
>>>>>>> f380bd248a44e99d1dc87085ebda5125608c9d1f
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>请输入保存的文件名</string>
     </property>
    </widget>
   </item>
<<<<<<< HEAD
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>华文细黑</family>
=======
   <item row="4" column="0">
    <widget class="QLineEdit" name="lineEdit_m3u8">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>请输入m3u8网址</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="pushButton_path">
     <property name="text">
      <string>选择下载路径</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>下载路径：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>m3u8请求URL：</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>保存文件名：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>华文楷体</family>
>>>>>>> f380bd248a44e99d1dc87085ebda5125608c9d1f
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>m3u8下载器</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
<<<<<<< HEAD
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButton_path">
     <property name="text">
      <string>选择下载路径</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>m3u8网址：</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>确认下载</string>
     </property>
    </widget>
   </item>
=======
>>>>>>> f380bd248a44e99d1dc87085ebda5125608c9d1f
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
